### DISCUSSION TITLE
Use on Windows Systems

### DISCUSSION THREAD

**Discussion by schiffinor** [rahulnyk/knowledge_graph#13] [Q&A]
Considering this is built around ollama I assume there is no alternative for windows systems, at least not with gpu integration. Or am I mistaken?

**Comment by rahulnyk** [comment]
you can probably use docker to host Ollama on windows. I have not tried it though. If you succeed please post the results here. 

Unfortunately Ollama doesn't support windows yet, But I think they will soon launch for windows.

**Reply by schiffinor** [reply]
btw got it to work using wsl (linux for windows) I'll explain in greater detail at some other point
